backend/wmg/config.py

```diff
@@ -7,19 +7,6 @@ class WmgConfig(SecretConfig):
     def __init__(self, *args, **kwargs):
         super().__init__("backend", secret_name="wmg_config", **kwargs)
 
-    # TODO: promote this impl to parent class, if new behavior works universally
```

Comment:
I would recommend putting this change and the update to `backend/corpora/common/utils/secret_config.py` in their own pr because the changes arent directly related to the gene info api